gensim/models/doc2vec.py

```diff
@@ -422,7 +422,7 @@ def init_sims(self, replace=False):
                     self.doctag_syn0norm = empty(self.doctag_syn0.shape, dtype=REAL)
                 np_divide(self.doctag_syn0, sqrt((self.doctag_syn0 ** 2).sum(-1))[..., newaxis], self.doctag_syn0norm)
 
-    def most_similar(self, positive=[], negative=[], topn=10, clip_start=0, clip_end=None, indexer=None):
+    def most_similar(self, positive=None, negative=None, topn=10, clip_start=0, clip_end=None, indexer=None):
```

Comment:
Making these defaults empty tuples – `()` – could enforce the immutability, and eliminate the later `None`-check/assignments.